### PR TITLE
fixing mpu6050 build

### PR DESCRIPTION
**Pull request by pelikhan**
MPU6050 not linking.

### CODE REVIEW

File: inc/drivers/MPU6050.h [hunk 1]
```diff
@@ -20,49 +20,12 @@ namespace codal{
         I2C& i2c;
         Pin             &int1;              // Data ready interrupt.
         uint16_t        address;            // I2C address of this accelerometer.
-        uint16_t        samplePeriod;       // The time between samples, in milliseconds.
```
Comment (pelikhan) [inline inc/drivers/MPU6050.h]: part of base class

File: inc/drivers/MPU6050.h [hunk 2]
```diff
@@ -20,49 +20,12 @@ namespace codal{
...
-        uint8_t         sampleRange;        // The sample range of the accelerometer in g.
-        Sample3D        sample;             // The last sample read.
         Sample3D        gyro;             // The gyro value
 
     public:
 
         MPU6050(I2C &_i2c, Pin &_int1, CoordinateSpace &coordinateSpace, uint16_t address = MPU6050_DEFAULT_ADDR,  uint16_t id = DEVICE_ID_ACCELEROMETER);
 
-        /**
```
Comment (pelikhan) [inline inc/drivers/MPU6050.h]: defined in base class

File: inc/drivers/MPU6050.h [hunk 3]
```diff
@@ -75,16 +38,6 @@ namespace codal{
           */
         int whoAmI();
 
-        /**
```
Comment (pelikhan) [inline inc/drivers/MPU6050.h]: defined in base class

File: inc/drivers/MPU6050.h [hunk 4]
```diff
@@ -130,11 +83,7 @@ namespace codal{
          */
         virtual int requestUpdate();
 
-        /**
-          * Destructor.
-          */
-        ~MPU6050();
-
+        virtual int setSleep(bool sleepMode);
```
Comment (pelikhan) [inline inc/drivers/MPU6050.h]: missing sleep api

File: source/drivers/MPU6050.cpp [hunk 5]
```diff
@@ -89,23 +89,17 @@ int MPU6050::updateSample()
...
 {
 
 }
 
-MPU6050::~MPU6050()
+int MPU6050::setSleep(bool sleepMode)
 {
-}
+    if (sleepMode)
+        return 0; // TODO
```
Comment (pelikhan) [inline source/drivers/MPU6050.cpp]: TODO, need to review the datasheet to implement sleep functionality.

Comment (mmoskal) [inline source/drivers/MPU6050.cpp]:
```suggestion
        return i2c.writeRegister(address, 0x6B, 0x40);
```

I think that should do the trick

**Pull request merged**